core/src/main/java/org/streamprocessor/core/transforms/SalesforceFn.java

```diff
@@ -103,14 +104,28 @@ public void processElement(@Element PubsubMessage received, OutputReceiver<Pubsu
                 out.output(
                         new PubsubMessage(payloadObject.toString().getBytes("UTF-8"), attributes));
             } catch (IllegalArgumentException e) {
-                LOG.error("IllegalArgumentException: ", e);
+                LOG.error(
+                        "exception[{}] step[{}] details[{}] entity[{}]",
+                        e.getClass().getName(),
+                        "DynamodbFn.processElement()",
+                        e.toString(),
+                        entity);
             } catch (org.json.JSONException e) {
-                LOG.error("org.json.JSONException: ", e);
+                LOG.error(
+                        "exception[{}] step[{}] details[{}] entity[{}]",
+                        e.getClass().getName(),
+                        "DynamodbFn.processElement()",
+                        e.toString(),
+                        entity);
             } catch (Exception e) {
-                LOG.error("Exception: ", e);
+                throw e;
```

Comment:
Do you need this:
`catch (Exception e) {
                throw e;`

Reply:
It will be catched either way by the catch on row 124?

Reply:
Thought so too but hesitated to remove the catch from existing code ^^